src/middlewares/error-handler.js

```diff
@@ -0,0 +1,17 @@
+const { INTERNAL_ERROR } = require('../utils/constants')
+const montarMensagemDeErroDeSchema = require('../utils/montarMensagemDeErroDeSchema')
+
+function errorHandler (error, _req, res, _next) {
+  const erroDeSchema = error.name === 'ValidationError'
+  /* istanbul ignore else */
+  if (erroDeSchema) {
+    return res.status(400).json(montarMensagemDeErroDeSchema(error))
+  } else if (error instanceof SyntaxError && error.status === 400) {
+    return res.sendStatus(400)
+  } else {
+    console.error(error)
+    return res.status(500).json({ message: INTERNAL_ERROR })
```

Comment:
Como precisamos dos dados do erro para que o usuário abra a issue, acho interessante retornar a mesma na resposta.

Isso vai facilitar na identificação do problema com o monitoramento feito no moesif.com

```suggestion
    return res.status(500).json({ message: INTERNAL_ERROR, error })
```